src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1157,6 +1159,8 @@ private async Task<ScriptInvocationResult> CreateScriptInvocationResult(Invocati
                 Return = invokeResponse?.ReturnValue?.ToObject()
             };
 
+            _workerChannelLogger.LogDebug("Successfully parsed the result of the invocation with invocationId: '{invocationId}'", invokeResponse.InvocationId);
```

Comment:
Added for testing purposes - most of the other unit tests for this class rely on logs (showing that function metadata has been processed, showing that invocations were received, etc), so followed the existing architecture.